### ISSUE TITLE
Wiki - Documentation Error with Access Example

### ISSUE THREAD

**Issue by iStryker** [RESTful-Drupal/restful#832]
@jeff-h want your input, just in case I am wrong.

https://github.com/RESTful-Drupal/restful/wiki/Custom-field-access-control

Should ACCESS_DENY, ACCESS_UNK, ACESS_GRANT be ACCESS_DENY, ACCESS_IGNORE, ACCESS_ALLOW?

**Comment by jeff-h** [comment]
Oh dear, you're right, but worse, that code's for v1.x. I've updated the page with a note to say as much, and fixed the constants — thanks for the heads-up!

Are you working in 1.x or 2.x? If the latter, any chance you'd mind creating a new version of that page for 2.x while your head's in the `access_callbacks` space?

**Comment by iStryker** [comment]
I am using 1.x.  I think the documenation is extremely simular....looking at 
https://github.com/RESTful-Drupal/restful/wiki/2.-Defining-the-public-fields-(2.x) and
https://github.com/RESTful-Drupal/restful/wiki/2.-Defining-the-public-fields-(1.x)

**Comment by jeff-h** [comment]
Yeah, the v2.x docs are mostly a clone of the 1.x with anything changed as necessary. In some cases not a lot changed, but what did was important enough to warrant the separate documentation. I am a little worried that some things have changed but not been reflected in the 2.x docs.

However, your discovery here was in fact wrong in both versions :)

**Comment by e0ipso** [comment]
Thanks @jeff-h and @iStryker. Please, close this issue when it's resolved.

**Comment by rhclayto** [comment]
It seems the documentation for custom field access control has been removed entirely. I just get 'Create new page' when I try to visit the URL. Is this documentation available somewhere else?
